src/modules/users/routes/user.route.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import { auth } from '../middlewares/auth';
+import { validation } from '../middlewares/validation';
+import { get, signup, signin } from '../controllers/user.controller';
+
+const userRouter = express.Router();
+
+userRouter.get('/', auth, get);
+userRouter.post('/signup', validation, signup);
+userRouter.post('/signin', auth, validation, signin);
```

Comment:
Do we need `auth` middleware for the signin? Since the user won't have the auth token, I think `auth` would not be required here.